internal/monitors/collectd/df/metadata.yaml

```diff
@@ -20,26 +20,32 @@ monitors:
       description: Number of inodes that are free.
       included: false
       type: gauge
+      group: reportInodes
```

Comment:
I just remembered, we decided on kebab case so these should be report-inodes and values-percentage.